x-pack/plugins/observability/public/pages/slos_page/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
I don't think we need `_page` in the name of the directory since it is already in the pages directory:
```
pages/slos_page/index.ts --> pages/slos/index.ts
```

Reply:
I agree with @maryam-saeidi 's suggestion

Reply:
I'll change that thanks!